core/new-gui/src/app/workspace/component/codearea-custom-template/codearea-custom-template.component.ts

```diff
@@ -53,6 +53,9 @@ export class CodeareaCustomTemplateComponent extends FieldType<any> implements A
    */
   onClickEditor(): void {
     this.dialogRef = this.dialog.open(CodeEditorDialogComponent, {
+      id: "mat-dialog-udf",
+      maxWidth: "95vw",
+      maxHeight: "95vh",
```

Comment:
same question here. why are we setting width and height here instead of in CSS?

Reply:
The max-width and max-height will not work in CSS. Seems like a limitation of the library. 
See the following:
https://stackoverflow.com/questions/46034619/angular-material-2-how-to-overwrite-dialog-max-width-to-100vw
https://github.com/angular/components/issues/3209